enzyme/Enzyme/TypeAnalysis/TypeTree.h

```diff
@@ -554,6 +554,8 @@ class TypeTree : public std::enable_shared_from_this<TypeTree> {
                 chunk = 8;
               } else if (flt->isHalfTy()) {
                 chunk = 2;
+              } else if (flt->isX86_FP80Ty()) {
```

Comment:
Also here

Reply:
also changed in [24c8098](https://github.com/EnzymeAD/Enzyme/pull/1604/commits/24c8098f99d83af45a0d681a44d9ec9411d40be3)